include/chopper/layout/configuration.hpp

```diff
@@ -37,6 +44,32 @@ struct configuration
     bool force_all_binnings{false};
     bool output_statistics{false};
     bool debug{false};
+
+private:
+    friend class cereal::access;
+
+    template <typename archive_t>
+    void serialize(archive_t & ar)
+    {
+        uint32_t version{1};
```

Comment:
I think the version should be a static constexpr member, and then we either write it out or check if the read version is equal to the member; just like in raptor.
Though, for now this is good enough, because we can change the handling afterwards.

Do you want to do it now or make it a task for later?

Reply:
task later :)